src/browserlib/extract-references.mjs

```diff
@@ -94,23 +100,45 @@ function nextTag(node, name) {
 function parseReferences(referenceList, options) {
   var defaultRef = [], informativeRef = [];
   options = options || {};
-  [].forEach.call(referenceList.querySelectorAll("dt"), function (dt) {
-    var ref = {};
-    ref.name = dt.textContent.replace(/[\[\] \n]/g, '');
-    var desc = nextTag(dt, "dd");
-    if (!desc || !ref.name) {
-      return;
-    }
-    const url = desc.querySelector('a[href*="://"]')?.href;
-    if (url) {
-      ref.url = url;
-    }
-    if (options.filterInformative &&
-        desc.textContent.match(/non-normative/i)) {
-      return informativeRef.push(ref);
-    }
-    defaultRef.push(ref);
-  });
+  if (referenceList.tagName === "DL") {
+    [...referenceList.childNodes]
+      .filter(child => child.tagName === "DT")
+      .forEach(function (dt) {
+        var ref = {};
+        ref.name = dt.textContent.replace(/[\[\] \n]/g, '');
+        var desc = nextTag(dt, "dd");
+        if (!desc || !ref.name) {
+          return;
+        }
+        const url = desc.querySelector('a[href*="://"]')?.href;
+        if (url) {
+          ref.url = url;
+        }
+        if (options.filterInformative &&
+            desc.textContent.match(/non-normative/i)) {
+          return informativeRef.push(ref);
+        }
+        defaultRef.push(ref);
+      });
+  }
+  else if (referenceList.tagName === "UL") {
+    [...referenceList.childNodes]
+      .filter(child => child.tagName === "LI")
+      .forEach(function (li) {
+        li = li.cloneNode(true);
+        [...li.querySelectorAll("ul")].map(el => el.remove());
```

Comment:
a comment as to why this is useful would be useful :)

Reply:
I expanded on why the code is not smart enough without that for [ECMA-402 references](https://tc39.es/ecma402/#normative-references).